packages/vue-app/template/index.js

```diff
@@ -123,7 +123,8 @@ async function createApp(ssrContext) {
     payload: ssrContext ? ssrContext.payload : undefined,
     req: ssrContext ? ssrContext.req : undefined,
     res: ssrContext ? ssrContext.res : undefined,
-    beforeRenderFns: ssrContext ? ssrContext.beforeRenderFns : undefined
+    beforeRenderFns: ssrContext ? ssrContext.beforeRenderFns : undefined,
```

Comment:
For the 5 lines above, would something like `...ssrContext` work or could that copy some variables over that were not intended to be there?

Reply:
I think the top-level ones were kept for backward-compatibility only. Spreading all of the `ssrContext` would introduce unwanted keys and perf overhead. I suggest to someday remove top level ones at all.